CMakeLists.txt

```diff
@@ -5,21 +5,7 @@ cmake_minimum_required(VERSION 3.26.4 FATAL_ERROR)
 project(
   pynvjitlink
   VERSION ${SKBUILD_PROJECT_VERSION}
-  LANGUAGES CXX CUDA
+  LANGUAGES CXX
 )
```

Comment:
Given that this PR turns pynvjitlink into a pure Python package (I think!), I don't really know what to put here, given that I've never used CMake to package a pure Python project... 😆

Reply:
I don't think it makes sense to involve CMake for a pure Python package. Probably only a pyproject.toml would be necessary.

Reply:
Yes, we would need to rearchitect the build systems and CI scripts to account for having only a single build of a pure Python package.